src/models/job.jsx

```diff
@@ -99,6 +99,10 @@ const JobContextProvider = ({ children }) => {
         id: doc.id,
         title: job.title,
         company,
+        status: job.status,
+        location: job.location,
+        wageHourlyMin: job.wageHourlyMin,
+        hasMedicalBenefit: job.hasMedicalBenefit,
```

Comment:
remove the two fields here

Reply:
replaced with new "wage" and "benefit" fields